lib/sealField.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const iron = require('iron');
+const nodeify = require('./nodeify');
+
+/**
+ * Seal a field
+ * @method sealField
+ * @param  {Object}         model               The model to seal a property of
+ * @param  {Object}         config
+ * @param  {Mixed}          config.field        The field to store the sealed value in
```

Comment:
I was thinking towards if we ever used it for other things in the future, but I can see how the logic about levels of indirection while debugging applies here too. Probably better to have it be more descriptive, and modify it if at some point in the future modifications are needed.